plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -164,9 +164,17 @@ class StmtConversionVisitor : FirVisitor<Exp, StmtConversionContext>() {
         return returnExp
     }
 
+    override fun visitImplicitInvokeCall(implicitInvokeCall: FirImplicitInvokeCall, data: StmtConversionContext): Exp {
+        val args = getFunctionCallArguments(implicitInvokeCall).map { it.accept(this, data) }
+        val retType = implicitInvokeCall.calleeReference.toResolvedCallableSymbol()!!.resolvedReturnType
+        val returnVar = data.newAnonVar(data.embedType(retType))
+        val returnExp = returnVar.toLocalVar()
+        data.addDeclaration(returnVar.toLocalVarDecl())
```

Comment:
Probably worth leaving a comment that we are intentionally not assigning to this variable.